diff --git a/BUILD.gn b/BUILD.gn
index 212ac82..6864bc1 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -11,7 +11,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import("//build/ohos.gni")
+import("//arkcompiler/runtime_core/gn/build/ohos.gni")
+
 
 THIRDPARTY_PROTOBUF_SUBSYS_NAME = "thirdparty"
 THIRDPARTY_PROTOBUF_PART_NAME = "protobuf"
@@ -50,9 +51,9 @@ ohos_shared_library("protobuf_lite") {
     "src/google/protobuf/stubs/time.cc",
     "src/google/protobuf/wire_format_lite.cc",
   ]
-  if (!is_asan && !is_debug) {
-    version_script = "libprotobuf_lite.map"
-  }
+  #if (!is_asan && !is_debug) {
+  #  version_script = "libprotobuf_lite.map"
+  #}
   include_dirs = [
     "src/google/protobuf/**/*.h",
     "src/google/protobuf/**/*.inc",
@@ -74,10 +75,10 @@ ohos_shared_library("protobuf_lite") {
 
   public_configs = [ ":protobuf_config" ]
   install_enable = true
-  innerapi_tags = [
-    "platformsdk_indirect",
-    "chipsetsdk",
-  ]
+  #innerapi_tags = [
+  #  "platformsdk_indirect",
+  #  "chipsetsdk",
+  #]
   subsystem_name = "${THIRDPARTY_PROTOBUF_SUBSYS_NAME}"
   part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
 }
@@ -121,7 +122,7 @@ ohos_static_library("protobuf_lite_static") {
     if (default_toolchain == current_toolchain) {
       # target build, not host build
       defines = [ "HAVE_HILOG" ]
-      external_deps = [ "hilog:libhilog" ]
+      #external_deps = [ "hilog:libhilog" ]
     }
   } else {
     defines = [ "_FILE_OFFSET_BITS_SET_LSEEK" ]
@@ -142,7 +143,7 @@ ohos_static_library("protobuf_lite_static") {
   }
   public_configs = [ ":protobuf_config" ]
   subsystem_name = "${THIRDPARTY_PROTOBUF_SUBSYS_NAME}"
-  part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
+  #part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
 }
 
 ohos_shared_library("protobuf") {
@@ -216,9 +217,9 @@ ohos_shared_library("protobuf") {
   ]
 
   deps = [ ":protobuf_lite" ]
-  if (!is_asan && !is_debug) {
-    version_script = "libprotobuf.map"
-  }
+  #if (!is_asan && !is_debug) {
+  #  version_script = "libprotobuf.map"
+  #}
 
   public_configs = [ ":protobuf_config" ]
   install_enable = true
@@ -300,7 +301,7 @@ ohos_static_library("protobuf_static") {
 
   public_configs = [ ":protobuf_config" ]
   subsystem_name = "${THIRDPARTY_PROTOBUF_SUBSYS_NAME}"
-  part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
+  #part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
 }
 
 if (current_toolchain == host_toolchain) {
@@ -525,7 +526,7 @@ if (current_toolchain == host_toolchain) {
 
     public_configs = [ ":protobuf_config" ]
     subsystem_name = "${THIRDPARTY_PROTOBUF_SUBSYS_NAME}"
-    part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
+    #part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
   }
 }
 
@@ -549,6 +550,6 @@ if (current_toolchain == host_toolchain) {
       "-Wno-deprecated-declarations",
     ]
     subsystem_name = "${THIRDPARTY_PROTOBUF_SUBSYS_NAME}"
-    part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
+    #part_name = "${THIRDPARTY_PROTOBUF_PART_NAME}"
   }
 }
